### PR TITLE
#432 Switch from Archiva to Bintray

### PR DESCRIPTION
**Pull request by jo-elimu**
- Updated Maven URL in README file

### CODE REVIEW

File: contentprovider/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Then, add the following dependencies:
 dependencies {
    ...
 
-   compile 'org.literacyapp:literacyapp-model:1.1.56'
+   compile 'org.literacyapp:literacyapp-model:1.1.67'
```
Comment (sladomic) [inline contentprovider/README.md]: I think you need to change this to "compile 'ai.elimu:model:1.1.67'" as shown under "Choose dependency snippet -> Gradle" https://bintray.com/elimu-ai/maven/model

File: contentprovider/build.gradle [hunk 2]
```diff
@@ -49,7 +49,7 @@ dependencies {
     })
     testCompile 'junit:junit:4.12'
 
-    compile 'org.literacyapp:literacyapp-model:1.1.56'
+    compile 'org.literacyapp:literacyapp-model:1.1.67'
```
Comment (sladomic) [inline contentprovider/build.gradle]: I think you need to change this to "compile 'ai.elimu:model:1.1.67'" as shown under "Choose dependency snippet -> Gradle" https://bintray.com/elimu-ai/maven/model

**Review by sladomic — CHANGES_REQUESTED** [review]

**Review by sladomic — APPROVED** [review]

File: contentprovider/README.md (hunk 1 above)
Comment (jo-elimu) [reply contentprovider/README.md]: Yes, you are right. Thanks for reminding me :-)

File: contentprovider/build.gradle (hunk 2 above)
Comment (jo-elimu) [reply contentprovider/build.gradle]: Fixed!

**Pull request merged**